release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -228,6 +228,18 @@ no operators found with name <name>.<version> in channel <name> of package <name
 
 === Operator development
 
+[id="ocp-4.8-sandboxed-containers"]
+=== {sandboxed-containers-first}
+
+[id="ocp-4.8-sandboxed-containers-tp"]
+==== {sandboxed-containers-first} support on {product-title} (Technology Preview)
+
+{sandboxed-containers-first} 1.0.0 Technology Preview release introduces built-in support for running Kata Containers as an additional runtime. {sandboxed-containers-first} provide users with the ability to choose from the type of workload that they want to run to cover a wide variety of use cases. You can use the {sandboxed-containers-operator} to perform tasks such as installation and removal, updates, and status monitoring.
+
+{sandboxed-containers-first} are only supported on bare metal. {op-system-first} is the only supported operating system for {sandboxed-containers-first} 1.0.0. Disconnected environments are not supported in {product-title} {product-version}.
```

Comment:
Why does `{product-title} {product-version}` show up as "OpenShift Container Platform Branch Build."? Are we making a support statement about OpenShift or about sandboxed containers?

Reply:
Branch Build will be 4.8. It's weird Preview link. When the doc goes live it will say 4.8.